recipes/fasttext/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "fasttext" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8c4ba14372d160fe7de902299caac664b7c1af1778601e17ca18ca2081b0e253" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 431c9e2.tar.gz
+  url: https://github.com/facebookresearch/fastText/archive/431c9e2.tar.gz
```

Comment:
`fasttext` did the github release in a very strange way, so this URL is kind of unfortunate. I think for increased maintainability, it'd be better to add `{% set release_name = "431c9e2" %}` under the `set version` line above, and then change this to `https://github.com/facebookresearch/fastText/archive/{{ release_name }}.tar.gz`.